src/main/java/net/dries007/tfc/api/capability/food/FoodStatsTFC.java

```diff
@@ -30,7 +30,8 @@
 @ParametersAreNonnullByDefault
 public class FoodStatsTFC extends FoodStats implements IFoodStatsTFC
 {
-    public static final float PASSIVE_HEAL_AMOUNT = 20 * 0.0002f; // On the display: 1 HP / 5 second
```

Comment:
These should all be reverted - they were part of a fix for nutrition decay that got fixed on the main branch instead